### PR TITLE
Fix plumbing constructor chemical grinders

### LINKED ISSUE #49026: Plumbing Constructor Chemical Grinders Broken
Issue reported from  Round ID: 128977 (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1507

The setup is shown. When plants go in the disposals chute their chems should enter the plumbing system. Currently the plant just disappears. No chems appear in the plumbing network and no plants leave the disposals system.

temp

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
Fixes #49026
:cl:
fix: grinder chem not grinding
fix: dir of grinder chem
/:cl:

### CODE REVIEW

File: code/modules/plumbing/plumbers/grinder_chemical.dm [hunk 1]
```diff
@@ -13,23 +13,10 @@
 	. = ..()
 	AddComponent(/datum/component/plumbing/simple_supply, bolt)
 
-/obj/machinery/plumbing/grinder_chemical/can_be_rotated(mob/user,rotation_type)
-	if(anchored)
```
Comment (ShizCalev) [inline code/modules/plumbing/plumbers/grinder_chemical.dm]: still need the anchored check.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by spookydonut** [comment]
Spurious travis failure

File: code/modules/plumbing/plumbers/grinder_chemical.dm [hunk 2]
```diff
@@ -13,23 +13,16 @@
...
-			eat_dir = EAST
-			return TRUE
-		if(SOUTH)
-			eat_dir = WEST
-			return TRUE
+	return TRUE
+
+/obj/machinery/plumbing/grinder_chemical/wrench_act(mob/living/user, obj/item/I)
+	..()
+	eat_dir = dir
```
Comment (ShizCalev) [inline code/modules/plumbing/plumbers/grinder_chemical.dm]: this should occur in setDir() instead

**Review by ShizCalev — DISMISSED** [review]

File: code/modules/plumbing/plumbers/grinder_chemical.dm [hunk 3]
```diff
@@ -7,29 +7,21 @@
...
-			eat_dir = EAST
-			return TRUE
-		if(SOUTH)
-			eat_dir = WEST
-			return TRUE
+	return TRUE
+
+/obj/machinery/plumbing/grinder_chemical/setDir(newdir)
+	. = ..()
+	eat_dir = dir
```
Comment (ShizCalev) [inline code/modules/plumbing/plumbers/grinder_chemical.dm]: eat_dir = newdir

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**